src/data/events/archive/winter2022/hoth-9.yml

```diff
@@ -0,0 +1,91 @@
+- name: Hack off the Hill 9 Workshops
+  quarter: Winter 2022
+  mainLink: https://www.youtube.com/watch?v=GskV5qMOpkE&list=PLPO7_kXilXFbjHembKB9UdcMyFkc_1Q06
+  tags: ["hackathon"]
+  directors:
+    - Christina Tong
+    - Einar Balan
+
+  workshops:
+    - name: Intro to Hackathons
+      youtube: https://www.youtube.com/watch?v=GskV5qMOpkE
+      tags: ["hackathon"]
+      presenters:
+        - Rajeshwari Jadhav
+    - name: Intro to HTML
+      youtube: https://youtu.be/qQy8lwOHkBY
+      repo: https://github.com/uclaacm/hoth8-intro-to-html-css-js
```

Comment:
You can link the individual sections from the README by clicking the link icon next to the header.

![image](https://user-images.githubusercontent.com/44387428/162851946-18a3d067-1aef-4557-9c69-19dfbba1dd9c.png)

This should give you a url with a tag like `#html` or `#css` appended to the end. When a user clicks on this link, the page will automatically scroll to that section.